### DISCUSSION TITLE
2.0.0

### DISCUSSION THREAD

**Discussion by volatile-static** [volatile-static/Chartero#57] [Announcements]
**Full Changelog**: https://github.com/volatile-static/Chartero/compare/1.3.3...2.0.0
[中文版更新说明](https://gitee.com/const_volatile/chartero/releases/tag/2.0.0)

<hr /><em>This discussion was created from the release <a href='https://github.com/volatile-static/Chartero/releases/tag/2.0.0'>2.0.0</a>.</em>

**Comment by alirezach** [comment]
I Cant install this. Zotero show error "The add-on "%S" could not be installed. It may be incompatible with this version of Zotero."

**Reply by volatile-static** [reply]
The v2.0 and above supports Zotero7 only. Please upgrade your Zotero.

**Comment by Echoiland** [comment]
respect！

**Comment by Echoiland** [comment]
是否没有一个一周总结的按钮了，在首页上面没有一个chartero的图标。

**Reply by volatile-static** [reply]
现在移动到主菜单里了
